### PR TITLE
Simplify motor idle config

### LINKED ISSUE #3571: Betaflight Configurator - confusing explaination for DShot/Dynamic Idle setting
### Describe the bug

Untitled

### To Reproduce

Under PID Tuning tab, found when hovering over the Dynamic Idle help button

### Expected behavior

A consistent use of either the term DShot Idle or Dynamic Idle if they mean the same thing.

And if the current version of the configurator is for firmware 4.4.x then, according to the pop-up, this setting is no longer relevant- implies that it no longer does anything- then the setting should be removed from the ui altogether.

If the terms DShot Idle and Dynamic Idle are indeed two distinct settings, direct the user to the other setting because I have no idea where it is.

### Configurator version

10.9.0

### Flight controller configuration

_No response_

### Add any other context about the problem that you think might be relevant here

_No response_

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes #3571

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
